src/components/ExtraPages/Help/DataHelp.js

```diff
@@ -1,35 +1,245 @@
+                    }>
+                    EuroCORDEX regional
+                  </HashLink>
+                </li>
+                <li>
+                  {" "}
+                  <HashLink
+                    to="/data-help#era"
+                    scroll={(el) =>
+                      el.scrollIntoView({ behavior: "auto", block: "center" })
+                    }>
+                    ERA-EuroCORDEX
+                  </HashLink>
+                </li>
+                <li>
+                  {" "}
+                  <HashLink
+                    to="/data-help#global"
+                    scroll={(el) =>
+                      el.scrollIntoView({ behavior: "auto", block: "center" })
+                    }>
+                    UKCP18 global
+                  </HashLink>
+                </li>
+                <li>
+                  {" "}
+                  <HashLink
+                    to="/data-help#regional"
+                    scroll={(el) =>
+                      el.scrollIntoView({ behavior: "auto", block: "center" })
+                    }>
+                    UKCP18 regional
+                  </HashLink>
+                </li>
+                <li>
+                  {" "}
+                  <HashLink
+                    to="/data-help#haduk"
+                    scroll={(el) =>
+                      el.scrollIntoView({ behavior: "auto", block: "center" })
+                    }>
+                    HadUK-Grid Observations
+                  </HashLink>
+                </li>
+                <li>
+                  {" "}
+                  <HashLink
+                    to="/data-help#other"
+                    scroll={(el) =>
+                      el.scrollIntoView({ behavior: "auto", block: "center" })
+                    }>
+                    Other
+                  </HashLink>
+                </li>
+              </ul>
+            </Nav>{" "}
+          </Container>
+        </Col>
+        {/* page contents */}
+        <Col>
+          <div>
+            <h1>Information on Data Sources</h1>
+            <p>
+              The main sources of climate information in this plot explorer are
+              the EuroCORDEX and UKCP18 regional ensembles, but several other
+              sources are also used: brief descriptions of these are provided
+              here. All indices are first computed on each model's native grid,
+              then{" "}
+              <a href="/indices-help">
+                regridded to a common 12km OSGB grid to facilitate comparisons.
+              </a>
+            </p>
+          </div>
+          <br></br>
+          {/* section about the data */}
+          <div id="cmip5-13">
+            <h5>CMIP5-13 ensemble</h5>
+            <p>
+              This consists of simulations from thirteen of the coupled
+              ocean-atmosphere models run as part of the Coupled Model
+              Intercomparison Project Phase 5 (CMIP5) experiment. This ensemble
```

Comment:
```suggestion
              ocean-atmosphere models run as part of the{" "}
              <a href="https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip5"> 
                 Coupled Model Intercomparison Project Phase 5 (CMIP5)
              </a>{" "}
              experiment. This ensemble
```

Reply:
@clairbarnes do all references to CMIP5 be links to https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip5 or just this first mention?

Reply:
Let's leave it as just this one for now. I can always add in more links later if we think it would be useful, but this is the most important place.